app/src/main/res/values/strings.xml

```diff
@@ -288,6 +288,12 @@
     <!-- Notification Channel Names -->
     <string name="notificationChannelTutorials">Tutorials</string>
 
+    <!-- AppTP Retention Experiment Notifications-->
+    <string name="enableAppTpNotificationEasyStepTitle">One easy step for app privacy</string>
+    <string name="enableAppTpNotificationEasyStepDescription">Enable App Tracking Protection to block trackers hiding in apps, even when you aren’t using your phone.</string>
```

Comment:
I'm not sure if this will cause any problems or not, but flagging for your thoughts:
- `aren’t`  - i don't know if that's the right character to use `’`
- elsewhere we use `\'`

Reply:
We use either of them from what I see. I always take the text with copy/paste to avoid any errors. In the past I asked this question and didn't get a compelling answer over which one to use.